src/tsconfig.monaco.json

```diff
@@ -8,7 +8,7 @@
 		],
 		"paths": {},
 		"module": "amd",
-		"moduleResolution": "classic",
+		"moduleResolution": "node",
```

Comment:
@hediet is this ok to do?

Reply:
Honestly, I have no idea what this changes. Why did you have to change it?

Reply:
The "Monaco Editor checks" fail with this if I don't change it:

![image](https://github.com/user-attachments/assets/48add612-bbd1-48d4-9f85-385617037da8)